spec/index.html

```diff
@@ -167,57 +167,71 @@ <h3>Resources and Statements</h3>
+      of a triple.</p>
 
-    <p>A <a>quoted triple</a> is not necessarily asserted, allowing
+    <p>The triples in a <a>graph term</a> are not necessarily asserted, allowing
       statements to be made about other statements that may not be
-      asserted within an <a>RDF graph</a>.
+      asserted within another <a>RDF graph</a>.
       This allows statements to be made about relationships that may be contradictory.
-      For a <a>quoted triple</a> to be asserted,
-      it must also appear in a graph as an <a>asserted triple</a>.</p>
+      For the triples in a <a>graph term</a> to be asserted,
+      they must also appear in a graph which is not a <a>graph term</a>.</p>
 
-    <p>The following diagram represents a statement with an unasserted <a>quoted triple</a>
+    <p>The following diagram represents a statement with an unasserted <a>graph term</a>
       as the subject.</p>
 
-    <figure id="fig-quoted-triple">
+    <figure id="fig-graph-term">
       <a href="quoted-triple.svg">
         <!-- Source for this file is at https://docs.google.com/drawings/d/1I_QxbUgnQXumXzb8c0WNJHIQ-mtRs2S80dDG6i9aOD8 -->
         <img src="quoted-triple.svg"
-             alt="An RDF graph containing a triple that references an unasserted quoted triple (with grey background) as the subject"
+             alt="An RDF graph containing a triple that references an unasserted graph term (with grey background) as the subject"
              style="width:70%"
-             aria-describedby="fig-quoted-triple-alt"/>
+             aria-describedby="fig-graph-term-alt"/>
       </a>
-      <figcaption id="fig-quoted-triple-alt">
-        An RDF graph containing a triple that references an unasserted quoted triple (with grey background) as the subject.
+      <figcaption id="fig-graph-term-alt">
+        An RDF graph containing a triple that references an unasserted graph term (with grey background) as the subject.
       </figcaption>
     </figure>
 
-    <p>A variation on the graph shown in <a href="#fig-quoted-triple"></a> can be described
-      where the <a>quoted triple</a> is also <a data-lt="asserted triple">asserted</a>.</p>
+    <p>A variation on the graph shown in <a href="#fig-graph-term"></a> can be described
+      where the triples in the <a>graph term</a> are also in the <a>default graph</a>.</p>
 
-    <figure id="fig-quoted-asserted-triple">
+    <figure id="fig-asserted-graph-term">
       <a href="quoted-asserted-triple.svg">
         <!-- Source for this file is at https://docs.google.com/drawings/d/1aoUEsb07P-nu5rvPRob7O07CRKgnf7tBw2CmRJNuaQ0 -->
         <img src="quoted-asserted-triple.svg"
-             alt="An RDF graph containing two triples where one of them contains the other as the subject; hence, that other triple is both quoted and asserted"
+             alt="An RDF graph containing two triples where one of them contains the other as the subject; hence, that other triple is both asserted and the content of a graph term"
```

Comment:
```suggestion
             alt="An RDF graph containing two triples, where one triple contains the second as its subject; hence, the second triple is both asserted _and_ contained within a graph term"
```

Reply:
That seems to use triples as resources, whereas one triple is both in the default graph and in a referenced graph term. Perhaps, something like the following:

```suggestion
             alt="An RDF graph containing two triples, where one triple has a subject which is a graph term containing the second triple; hence, the second triple is both asserted _and_ contained within a graph term"
```

Reply:
Works for me.